context/app/static/js/components/savedLists/AddToList/AddToList.jsx

```diff
@@ -13,11 +13,12 @@ function AddToList({ selectedLists, addToSelectedLists, removeFromSelectedLists
 
   return (
     <MaxHeightList>
-      {Object.keys(savedLists).map((listName) => (
+      {Object.entries(savedLists).map(([listUuid, value]) => (
         <AddToListItem
-          key={listName}
-          title={listName}
-          isSelected={selectedLists.has(listName)}
+          key={listUuid}
+          title={value.title}
+          listUuid={listUuid}
+          isSelected={selectedLists.has(listUuid)}
```

Comment:
```suggestion
          key={listUUID}
          title={value.title}
          listUuid={listUUID}
          isSelected={selectedLists.has(listUUID)}
```
(I won't comment on every occurrence below.)